### DISCUSSION TITLE
Is it possible to use a key.map as defined by 'bubbles/help' as keymap?

### DISCUSSION THREAD

**Discussion by MaximilianSoerenPollak** [Evertras/bubble-table#183] [Q&A] [answered]
As stated in the title I'm currently wanting to render the 'help' for the Table controlls. I do this via: 
`bubbles/help` but that one requires the keymapping to be fullfilling the key.Map interface. 

Is there a way easily to integrate those keybinds into this table? 
Or would be adding all commands in the update the only way?

**Comment by MaximilianSoerenPollak** [comment]
I could make a PR that enables the KeyMap to rather doable. Since it only requires two things to fullfill this. 
See:
[source file](https://github.com/charmbracelet/bubbles/blob/4382fdf1c6cfe18d34615c1e8b94b068022a461a/help/help.go#L17)

```go
type KeyMap interface {
	// ShortHelp returns a slice of bindings to be displayed in the short
	// version of the help. The help bubble will render help in the order in
	// which the help items are returned here.
	ShortHelp() []key.Binding

	// FullHelp returns an extended group of help items, grouped by columns.
	// The help bubble will render the help in the order in which the help
	// items are returned here.
	FullHelp() [][]key.Binding
}
```

I could implement that quickly if it was wished.

**Comment by Evertras** [comment, marked as answer]
For tracking, moving to https://github.com/Evertras/bubble-table/issues/184
